examples/easy_lend/tsconfig.json

```diff
@@ -9,7 +9,9 @@
     "strict": true,
     "esModuleInterop": true,
     "resolveJsonModule": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "rootDir": "src/",
+    "outDir": "./tslib"
```

Comment:
why here the output directory is `tslib` but in `sdk/js/tsconfig.json` it is `lib`?

Reply:
forge dependencies are by default in the lib folder. So we chose a different folder name to avoid any conflicts :D
I will add a comment for it

Reply:
ok, it's a json file so I can't add a comment :))